enhancements/machine-api/machine-api-usage-telemetry.md

```diff
@@ -0,0 +1,338 @@
+(for example MachineSets) I will learn about usage patterns and trends. Learning
+more about how frequently these components are deployed and utilized will allow me to
+identify areas where users might be having difficulties, make better decisions
+about the future planning of work, and demonstrate to a wider audience the popularity
+level of various features.
+
+#### Story 2
+
+As an OpenShift developer I would like to gain a deeper understanding of the
+general health of clusters by examining the behavior of the Machine Health
+Check. By analyzing the frequency of success and failures rates as well as
+the operational modes of the Machine Health Check I will be able to build
+a model of infrastructure health and potential predictive activity. This
+information will allow me to gain insight into correlations between
+unhealthy cluster remediations and infrastructure providers, give a window
+into predictive possibilities for user outreach, and demonstrate the value of
+automated remediation.
+
+#### Story 3
+
+As an OpenShift developer I would like to correlate Machine API resource usage
+with alerting events to provide presciptive advice about failures conditions
+and platform instability. By examining data about Machine API resource management,
+failures, and remediations, and corelating that data with alerts and platform
+information I will be able to better advise users about errors and methods for
+avoiding them. When combined with a wider fleet view this information will help
+me to create advisories about provider instabilities and potential outtages.
+
+### Implementation Details/Notes/Constraints
+#### Specific Telemetry Inquiries
+
+This section details some of the specific use cases that have been identified
+as initial inquries we would like to gather and report on. They have been
+categorized into 3 epics: demonstration of value, engineering insights,
+and proactive support. Demonstration of value speaks to the raw amount of
+usage that is happening for any given component. Engineering insights speaks
+to situations where we can learn about user experience and interactions to inform
+future planning and guidance. Proactive support speaks to information that we
+learn which will lead to better experiences for users by providing opportunity
+for preventitive intervention.
+
+**Total Remediation Events**
+
+Epics: Demonstration of Value, Proactive Support
+
+This metric will give us insight into how effectively the MHC is being used.
+A sustained high number of events on a particular cluster will enable us to
+deliver proactive support. A sudden spike across all clusters might give us
+early warning to a cloud provider’s data center outage, or a large network
+or similar event that is impacting many of our users.
+
+For example, since many clusters operate at a relatively small scale, it’s
+possible that losing an AZ in US-East-1 would be small enough event on their
```

Comment:
Might be good to clarify you mean the AWS region for those who aren't spending every day working with clouds
```suggestion
possible that losing an AZ in AWS US-East-1 would be small enough event on their
```